.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -85,6 +85,32 @@ runs:
       if [[ "$GIT_REF" == '' ]]; then 
         GIT_REF=$(git rev-parse HEAD)
       fi
+
+      if [[ "$K8S_VERSION" == '1.23' ]]; then 
```

Comment:
Comment here on what this is doing so it's really easy to quickly parse through what this if/else is doing?